index.html

```diff
@@ -2086,8 +2086,7 @@ <h3>Retrieve Verification Method</h3>
 The following algorithm specifies how to safely retrieve a verification method,
 such as a cryptographic [=public key=], by using a [=verification method=]
 identifier. Required inputs are a
-<strong>[=verification method=]</strong> (<var>verificationMethod</var>),
-a <strong>proof purpose</strong> (<var>proofPurpose</var>), and
```

Comment:
This shouldn't be removed, but perhaps should be changed to "verification relationship" (which can be specified as a `proofPurpose` in securing formats). It could also be made optional and default to `verificationMethod`.

Reply:
@dlongley can you please suggest a specific text change for us to consider doing what you propose?  Thanks.